libpod/container_internal.go

```diff
@@ -1378,6 +1378,17 @@ func (c *Container) addImageVolumes(ctx context.Context, g *generate.Generator)
 	if err != nil {
 		return err
 	}
+	// Add the built-in volumes of the container passed in to --volumes-from
```

Comment:
This probably doesn't end up being a part of WithImageVolumes

Reply:
Or maybe we rename addImageVolumes to `addLocalVolumes()`... That sounds good,

Reply:
Fixed